File: Exercises_C++/1012.cpp
#include <iostream>
#include <iomanip>
using namespace std;

int main(){

    float a;
    float b;
    float c;
    cin >> a >> b >> c;

    double triangulo;
    triangulo = (a * c) / 2;

    double circulo;
    circulo = 3.14159 * c * c;

    double trapezio;
    trapezio = ((a + b) * c) / 2;

    double quadrado;
    quadrado = b * b;

    double retangulo;
    retangulo = a * b;

    cout << fixed << setprecision(3);
    cout << "TRIANGULO: " << triangulo << endl;
    cout << "CIRCULO: " << circulo << endl;
    cout << "TRAPEZIO: " << trapezio << endl;
    cout << "QUADRADO: " << quadrado << endl;
    cout << "RETANGULO: " << retangulo << endl;

}

File: Bee-Crowd/Exercises_Python/1012.py
a, b, c = input().split()

a = float(a)
b = float(b)
c = float(c)


print('TRIANGULO: {:.3f}'.format((a * c) / 2))
print('CIRCULO: {:.3f}'.format(3.14159 * c * c))
print('TRAPEZIO: {:.3f}'.format(((a + b) * c) / 2))
print('QUADRADO: {:.3f}'.format(b * b))
print('RETANGULO: {:.3f}'.format(a * b))